auth-web/src/locales/en.json

```diff
@@ -4,6 +4,7 @@
   "techSupportPhone": "250 952-0568",
   "techSupportEmail": "SBC_ITOperationsSupport@gov.bc.ca",
   "payFailedMessage": "Payment Failed for technical reasons.Please revisit the site",
+  "payFailedMessagePayBcDown": "Payment Failed for technical reasons.Please use the below contacts",
```

Comment:
A minor copy correction: should be a single space after the `.` at the end of the sentence.  Same for the `payFailedMessage`.

Reply:
Changed.